src/pymor/models/loader.py

```diff
@@ -0,0 +1,116 @@
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+
+import yaml
```

Comment:
Practically this is a new (optional)  dependency with `pyyaml`. Can you add this to the config mechanism, `dependencies.py` and then `config.require('PYYAML')` it here?

Have you actively decided for `pyyaml` as opposed to `ruamel.yaml`? I understand the former doesn't understand the current YAML spec, while the latter doesn't. I cannot say however if we should care about this.

Reply:
I think we should use `pyyaml` as the [standard implementation](https://yaml.org/spec/1.2.2/#12-yaml-history) with a very active community. I do not think the changes in the standard affect the use case here. And the `pyyaml` seems to be [working on implementing the current standard](https://github.com/yaml/pyyaml/pull/555). However, I will document that we expect the standard `1.1`.

Reply:
Sounds good :+1: